pyfolio/perf_attrib.py

```diff
@@ -17,13 +17,25 @@
 import empyrical as ep
 import pandas as pd
 
-from itertools import chain
 import matplotlib.pyplot as plt
 from pyfolio.pos import get_percent_alloc
-from pyfolio.utils import print_table
-
-
-def perf_attrib(returns, positions, factor_returns, factor_loadings):
+from pyfolio.utils import print_table, set_legend_location
+
+# 31 visually distinct colors
+# http://phrogz.net/css/distinct-colors.html
+COLORS = [
+    '#f23d3d', '#828c23', '#698c83', '#594080', '#994d4d',
+    '#206380', '#dd39e6', '#cc9999', '#7c8060', '#66adcc',
+    '#6c7dd9', '#8a698c', '#7f6340', '#66cc7a', '#a3abd9',
+    '#d9c0a3', '#bfffcc', '#542699', '#b35986', '#d4e639',
+    '#b380ff', '#e0e6ac', '#a253a6', '#418020', '#ff409f',
+    '#ffa940', '#83ff40', '#3d58f2', '#e3ace6', '#d9a86c',
+    '#2db391'
+]
```

Comment:
Would it make sense to put this in `pyfolio.utils`?

Reply:
yup, I'll move it there